jerry-core/ecma/builtin-objects/ecma-builtin-bigint-prototype.c

```diff
@@ -101,21 +107,21 @@ ecma_builtin_bigint_prototype_object_to_string (ecma_value_t this_arg, /**< this
 
     if (ECMA_IS_VALUE_ERROR (ecma_op_to_integer (arguments_list_p[0], &arg_num)))
     {
-      ecma_free_value (bigint);
```

Comment:
As the "this_arg" is an input argument, it should not be freed here. But at the caller function.